data/cameras.xml

```diff
@@ -8598,6 +8598,19 @@
 		<Crop x="0" y="0" width="-128" height="0"/>
 		<Sensor black="1024" white="16383"/>
 	</Camera>
+	<Camera make="FUJIFILM" model="X-Pro2" mode="compressed">
+		<ID make="Fujifilm" model="X-Pro2">Fujifilm X-Pro2</ID>
+		<CFA2 width="6" height="6">
+			<ColorRow y="0">GGRGGB</ColorRow>
+			<ColorRow y="1">GGBGGR</ColorRow>
+			<ColorRow y="2">BRGRBG</ColorRow>
+			<ColorRow y="3">GGBGGR</ColorRow>
+			<ColorRow y="4">GGRGGB</ColorRow>
+			<ColorRow y="5">RBGBRG</ColorRow>
+		</CFA2>
+		<Crop x="0" y="6" width="-16" height="0"/>
```

Comment:
@muessel 
An entry for FUJIFILM X100F is missing.
Is there some problem with that camera, or it is a simple oversight?